src/components/Footer/Footer.module.css

```diff
@@ -1,31 +1,29 @@
 footer {
-    position: fixed;
+  /* position: fixed;
```

Comment:
Did you remove position fixed to footer?

Reply:
Yes. I think It will be better